app/src/main/java/com/spendster/presentation/validation/ComplexPhoneValidator.java

```diff
@@ -15,14 +15,6 @@ public ValidationResource validate() {
             errorMessage = "First and Last name field is empty";
             isValid = false;
         }
-        else if (this.phone.length() < 10){
```

Comment:
@SimonMaha we should definitely add phone validation, but it should be correct. For example, https://www.journaldev.com/641/regular-expression-phone-number-validation-in-java